### ISSUE TITLE
[Feature Request] Post Image URL

### ISSUE THREAD

**Issue by roxx3r** [diezo/Ensta#74]
Thanks so much for providing this code. In the past Instagram endpoints have provided an image URL or thumbnail for posts. Is that available from the API you're using?

**Comment by diezo** [comment]
yes that's definitely available on the API that ensta uses. i'll push an update for this and let you know.

also, please do give this repository a star so that other developers can know about ensta as well.
thanks
